source/playbooks/playbook-permissions.rst

```diff
@@ -0,0 +1,78 @@
+Roles and Permissions
+=====================
+
+There are different ways to access and interact with playbooks and this is controlled in the System Console in the form of permissions. Permissions can be granted in a variety of ways, to allow for different combinations of access and visibility, and to different roles. Two permission schemes are provided in Mattermost:
+
+* **System Scheme:** Applies permissions universally across all teams and channels.
+* **Team Override Schemes:** Allow admins to customize permissions for each team (available in Mattermost Enterprise and Professional).
+
+You can set the default permissions granted to System Admins, Team Admins, Channel Admins, Playbook Admins, Guests (if enabled), and All Members. The permissions granted in the System Scheme apply system-wide, meaning:
+
+* **Guests:** If Guest Accounts are enabled, permissions apply to guest users in all channels, in all teams.
+* **All Members:** Permissions apply to all members, including Admins, in all channels, in all teams.
+* **Channel Administrators:** Permissions apply to all Channel Admins in all channels, in all teams.
+* **Playbook Admins:** Permissions apply to all Playbook Admins in all channels, in all teams.
+* **Team Administrators:** Permissions apply to all Team Admins, in all teams.
+
+For more information about System and Team Override Schemes, refer to the `Advanced Permissions <https://docs.mattermost.com/onboard/advanced-permissions.html>`__ documentation.
+
+Playbooks roles
+---------------
+
+**Member**
+
+In the context of Playbooks, members are users of Mattermost who belong to a playbook or are a Playbook Admin.
+
+**Participant**
+
+In the context of Playbooks, participants belong to or take part in a run. They can be a member or a Playbook Admin.
```

Comment:
@crspeller, I'm hitting a blank: Do participants have to be part of a playbook or can they participate in a run without being a member of a playbook?

Reply:
The participants list in the RHS is all the members of the run's channel excluding the owner and any bots.

Reply:
Good point. I was trying to work the definitions into the documentation and this was a late addition. Happy to remove.